ts/features/pn/aar/store/reducers/index.ts

```diff
@@ -38,3 +39,11 @@ export const currentAARFlowData = (state: GlobalState) =>
   state.features.pn.aarFlow;
 export const currentAARFlowStateType = (state: GlobalState) =>
   state.features.pn.aarFlow.type;
+export const currentAARFlowStateErrorCodes = (state: GlobalState) => {
+  const aarFlow = state.features.pn.aarFlow;
+  if (aarFlow.type === sendAARFlowStates.ko) {
+    return aarFlow.error?.errors?.map(x => x.code) ?? [];
+  } else {
+    return [];
```

Comment:
Returning an empty array causes the selector's output to always be a new instance, this providing additional re-renderings. Declare and reference a constant empty array at file level instead.